src/machine/machine_avr.go

```diff
@@ -6,6 +6,8 @@ import (
 	"device/avr"
 )
 
+const CPU_FREQUENCY = 16000000
```

Comment:
Can you move this to machine_arduino.go? This value is board-specific, and should be defined separately for each board (for example, some Arduino variants like the [SparkFun Pro Micro](https://www.sparkfun.com/products/12587) run at 8MHz).

Reply:
I think you mean `board_arduino.go`, and done.